cloudgenix_config/pull.py

```diff
@@ -1240,6 +1274,31 @@ def _pull_config_for_single_site(site_name_id):
         # ensure the snmp config is not completely empty
         delete_if_empty(element, 'snmp')
 
+        # Get DNS configs
+        element[DNS_SERVICES_STR] = {}
+        response = sdk.get.dnsservices(site['id'], element['id'])
+        if not response.cgx_status:
+            throw_error("DNS config get failed: ", response)
```

Comment:
Can we change the error statement to reflect the API call since we have several DNS "config" related calls. 
Change to: "DNS Services get failed"
I also noticed some typos w.r.t throw_error messages in _pull_config_for_single_site when calls are made for hubcluster (673), spokeclusters(696) and dhcpservers(716)

Reply:
I have made the changes for the same and pushed the code